internal/kafka/internal/workers/cluster_mgrs/clusters_mgr.go

```diff
@@ -268,12 +276,16 @@ func (c *ClusterManager) processProvisionedClusters() []error {
 
 	// process each local provisioned cluster and apply necessary terraforming.
 	for _, provisionedCluster := range provisionedClusters {
-		glog.V(10).Infof("provisioned cluster ClusterID = %s", provisionedCluster.ClusterID)
-		metrics.UpdateClusterStatusSinceCreatedMetric(provisionedCluster, api.ClusterProvisioned)
-		err := c.reconcileProvisionedCluster(provisionedCluster)
-		if err != nil {
-			errs = append(errs, errors.Wrapf(err, "failed to reconcile provisioned cluster %s", provisionedCluster.ClusterID))
-			continue
+		if provisionedCluster.ClusterType != api.Enterprise.String() {
+			glog.V(10).Infof("provisioned cluster ClusterID = %s", provisionedCluster.ClusterID)
+			metrics.UpdateClusterStatusSinceCreatedMetric(provisionedCluster, api.ClusterProvisioned)
+			err := c.reconcileProvisionedCluster(provisionedCluster)
+			if err != nil {
+				errs = append(errs, errors.Wrapf(err, "failed to reconcile provisioned cluster %s", provisionedCluster.ClusterID))
+				continue
+			}
+		} else {
+			glog.V(10).Infof("skipping reconciliation of provisioned %s cluster with ClusterID = %s", api.Enterprise.String(), provisionedCluster.ClusterID)
```

Comment:
let's take the comment in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1429/files#r1039519780 onto account